locales/en.json

```diff
@@ -1,71 +1,77 @@
-  "Language name must contain letters only": "Language name must contain letters only",
-  "Language name must be atleast {#limit} characters long": "Language must be atleast {#limit} characters long",
-  "Language name  required": "Language is required",
-  "Country must contain letters only": "Country must contain letters only",
-  "Please fill in your country": "Please fill in your country",
-  "Country is required": "Country is required",
-  "Currency must contain letters only": "Currency must contain letters only",
-  "Please fill in your currency": "Please fill in your currency",
-  "Currency is required": "Currency is required",
-  "Department must contain letters only": "Department must contain letters only",
-  "Please fill in your department": "Please fill in your department",
-  "Department must be atleast {#limit} characters long": "Department must be atleast {#limit} characters long",
-  "Department is required": "Department is required",
-  "Line manager must contain letters only": "Line manager must contain numbers only",
-  "Please fill in the line manager": "Please fill in the line manager",
-  "Line manager must be atleast {#limit} characters long":"Line manager must be atleast {#limit} characters long",
-  "Line manager is required": "Line manager is required",
-  "User not found": "User not found",
-  "Profile is already complete": "Profile is already complete",
-  "Profile completed succesfuly": "Profile completed succesfuly",
-  "Profile updated succesfuly": "Profile updated succesfuly"
+	"Welcome to barefoot nomad": "Welcome to barefoot nomad",
+	"Email already exists": "Email already exists",
+	"Successfully registered": "Successfully registered",
+	"Email or password is incorrect": "Email or password is incorrect",
+	"Your are successfully loged in": "Your are successfully loged in",
+	"First name must be string": "First name must be string",
+	"Please fill in your first name": "Please fill in your first name",
+	"First name must contain letters only": "First name must contain letters only",
+	"First name must be atleast {#limit} characters long": "First name must be atleast {#limit} characters long",
+	"First Name is required": "First Name is required",
+	"Last name must be string": "Last name must be string",
+	"Please fill in your last name": "Please fill in your last name",
+	"Last name must contain letters only": "Last name must contain letters only",
+	"Last name must be atleast {#limit} characters long": "Last name must be atleast {#limit} characters long",
+	"Last Name is required": "Last Name is required",
+	"Email name must be string": "Email name must be string",
+	"Please fill in your email": "Please fill in your email",
+	"email must be atleast {#limit} characters long": "email must be atleast {#limit} characters long",
+	"email is required": "email is required",
+	"Phone Number contain numbers": "Phone Number contain numbers",
+	"Please fill in your Phone Number": "Please fill in your Phone Number",
```

Comment:
Remember we are using 2 spaces. Please check prettier config to ensure you put the correct spacing

Reply:
Okay